### PR TITLE
Add skeleton dropwizard app

### PR DESCRIPTION
**Pull request by gidsg**
Doesn't do much but should start up, render page at `/` and have healthcheck.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        </dependency>
+        <dependency>
+            <groupId>jakarta.validation</groupId>
+            <artifactId>jakarta.validation-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
+            <artifactId>hibernate-validator</artifactId>
+        </dependency>
+    </dependencies>
```
Comment (alexbishop1) [inline pom.xml:52]: I think we want to set explicit versions for these so we can control the updates.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>3.2.4</version>
+                <configuration>
```
Comment (alexbishop1) [inline pom.xml:59]: In other POMs we have `<outputFile>target/${project.artifactId}-${project.version}-allinone.jar</outputFile>` — do we want something similar here to be consistent? We have to reference the file name in the Dockerfile and possibly other places.

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>3.0.0-M5</version>
+            </plugin>
+        </plugins>
+    </build>
+
+    <reporting>
+        <plugins>
+        </plugins>
+    </reporting>
```
Comment (alexbishop1) [inline pom.xml:117]: Do we need this section if it’s empty?

File: src/main/java/uk/gov/pay/webhooks/WebhooksApp.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+public class WebhooksApp extends Application<WebhooksConfiguration> {
+    public static void main(String[] args) throws Exception {
+        new WebhooksApp().run(args);
+    }
+
+@Override
+public void run(WebhooksConfiguration configuration,
+                Environment environment) {
+    System.out.println("hello world");
```
Comment (alexbishop1) [inline src/main/java/uk/gov/pay/webhooks/WebhooksApp.java:17]: Not sure we want to leave this in.

File: src/main/java/uk/gov/pay/webhooks/healthcheck/HealthCheckResource.java [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    @Inject
+    public HealthCheckResource(Environment environment) {
+        this.environment = environment;
+    }
+
+    @GET
+    @Path("/")
+    @Produces(APPLICATION_JSON)
+    public Response healthCheck() {
+        var ok = ImmutableMap.of("healthy", "true");
```
Comment (alexbishop1) [inline src/main/java/uk/gov/pay/webhooks/healthcheck/HealthCheckResource.java:27]: We can use `Map.of("healthy", "true")` here.

File: src/main/resources/banner.txt [hunk 6]
```diff
@@ -0,0 +1,10 @@
+       ____
+      / __ \____ ___  __
+     / /_/ / __ `/ / / /
+    / ____/ /_/ / /_/ /
+   /_/    \__,_/\__, /
+               /____/
```
Comment (alexbishop1) [inline src/main/resources/banner.txt:6]: Snazzy!

**Review by alexbishop1 — CHANGES_REQUESTED** [review]
Looks like basically what we want.

File: src/main/java/uk/gov/pay/webhooks/WebhooksApp.java (hunk 4 above)
Comment (gidsg) [reply src/main/java/uk/gov/pay/webhooks/WebhooksApp.java]: removed

File: src/main/java/uk/gov/pay/webhooks/healthcheck/HealthCheckResource.java (hunk 5 above)
Comment (gidsg) [reply src/main/java/uk/gov/pay/webhooks/healthcheck/HealthCheckResource.java]: done

**Comment by gidsg** [comment]
@alexbishop1 I have force-pushed to address your comments and made a couple of other tweaks in separate comments.

File: pom.xml (hunk 2 above)
Comment (gidsg) [reply pom.xml:54]: done

File: pom.xml (hunk 1 above)
Comment (gidsg) [reply pom.xml]: have added explicit versions and removed the ones we aren't currently using.

**Review by alexbishop1 — APPROVED** [review]
This will do for now.

**Pull request merged**